src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1074,4 +1074,22 @@
   <data name="JsonNetMaybeInsecureSerializerTitle" xml:space="preserve">
     <value>Ensure that JsonSerializer has a secure configuration when deserializing</value>
   </data>
+  <data name="UseDefaultDllImportSearchPathsAttribute" xml:space="preserve">
+    <value>Use DefaultDllImportSearchPaths attribute for P/Invokes</value>
+  </data>
+  <data name="UseDefaultDllImportSearchPathsAttributeDescription" xml:space="preserve">
+    <value>By default, P/Invokes using DllImportAttribute probe a number of directories, including the current working directory for the library to load. This can be a security issue for certain applications, leading to DLL hijacking.</value>
+  </data>
+  <data name="UseDefaultDllImportSearchPathsAttributeMessage" xml:space="preserve">
+    <value>The method {0} didn't use DefaultDllImportSearchPaths attribute for P/Invokes.</value>
+  </data>
+  <data name="DoNotUseUnsafeDllImportSearchPath" xml:space="preserve">
+    <value>Do not use unsafe DllImportSearchPath value</value>
+  </data>
+  <data name="DoNotUseUnsafeDllImportSearchPathDescription" xml:space="preserve">
+    <value>There could be malicious DLL under the application directory or the default DLL search directories. Use an DllImportSearchPath value that sepecifies explicit search path instead</value>
+  </data>
+  <data name="DoNotUseUnsafeDllImportSearchPathMessage" xml:space="preserve">
+    <value>Use unsafe DllImportSearchPath value {0}</value>
```

Comment:
>Use unsafe DllImportSearchPath value {0} [](start = 11, length = 40)

Use a safe DllImportSearchPath value.

(I don't think we need to point out the integer value.)